front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -39,11 +39,10 @@ export default function SpeedLimitByTagSelector({
       >
         <img width="32px" src={icon} alt="speedometer" />
         <span className="text-muted">{t('speedLimitByTag')}</span>
-        <SelectImprovedSNCF
-          sm
+        <Select
+          small
```

Comment:
In the mockup, the component seems to have the same font size as the rest of the form

```suggestion
```